src/main/java/seedu/address/logic/parser/DoneCommandParser.java

```diff
@@ -0,0 +1,46 @@
+package seedu.address.logic.parser;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_DATE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TIME;
+
+import seedu.address.logic.commands.DateTimeLoader;
+import seedu.address.logic.commands.DoneCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.appointment.Date;
+import seedu.address.model.appointment.Time;
+
+/**
+ * Parses input arguments and creates a new DoneCommand object
+ */
+public class DoneCommandParser implements Parser<DoneCommand> {
+    /**
+     * Parses the given {@code String} of arguments in the context of the DoneCommand
+     * and returns an DoneCommand object for execution.
+     * @throws ParseException if the user input does not conform the expected format
```

Comment:
Think you missed out a "to" and full stop here. Perhaps it will be better to phrase it as `if the user input does not conform to the expected format.`